tests/Microsoft.Bot.Builder.TestBot.Json/CustomSteps/MultiplyDialog.cs

```diff
@@ -50,8 +50,8 @@ public MultiplyDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
 
         public override Task<DialogTurnResult> BeginDialogAsync(DialogContext dc, object options = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            var (arg1, err1) = Expression.Parse(Arg1).TryEvaluate(dc.GetState());
-            var (arg2, err2) = Expression.Parse(Arg2).TryEvaluate(dc.GetState());
+            var (arg1, err1) = new ExpressionParser().Parse(Arg1).TryEvaluate(dc.GetState());
```

Comment:
>new ExpressionParser( [](start = 31, length = 21)

ExpressionBuilder.Parse() please

Reply:
is expressionPraser.parse more consistent lgParser.parse?

Reply:
No, I prefer Expression.Parse(Arg1).TryEvaluate()
It just makes the concept count lower if you are looking at this as a sample

---
In reply to: [386753486](https://github.com/microsoft/botbuilder-dotnet/pull/3471#discussion_r386753486) [](ancestors = 386753486)